src/rtc/rtc_s1.py

```diff
@@ -74,18 +74,21 @@ def split_runconfig(cfg_in,
 
     # determine the scratch path for the child process
 
-    if not scratch_path_child:
-        scratch_path_child = \
+    if not child_scratch_path:
+        child_scratch_path = \
             os.path.join(cfg_in.groups.product_group.scratch_path,
-                         f'{os.path.basename(output_dir_child)}_child_scratch')
+                         f'{os.path.basename(child_output_dir)}_child_scratch')
 
     # determine the output directory for child process
+    basename_logfile = os.path.basename(parent_logfile_path)
     for burst_id in list_burst_id:
         path_temp_runconfig = os.path.join(cfg_in.scratch_path,
                                            f'burst_runconfig_{burst_id}.yaml')
-        if path_parent_logfile:
-            path_logfile_child = os.path.join(output_dir_child,
-                                              f'{burst_id}.{path_parent_logfile}')
+        if parent_logfile_path:
+            path_logfile_child = os.path.join(child_output_dir,
+                                              burst_id,
+                                              f'{burst_id}.{basename_logfile}')
```

Comment:
```suggestion
                                              f'{basename_logfile}')
```
Maybe we don't need the f'{burst_id}.` prefix? 

Not for this PR, but I'm thinking now.. another option would be to use the `product_id` here as we do for the other RTC-S1 layers. Then at the end we concatenate all burst logs into the log file that the user set in the CLI command.

Reply:
I don't have a strong opinion regarding the names for the per-burst log file, so I'll get rid of the prefix.

>Not for this PR, but I'm thinking now.. another option would be to use the product_id here as we do for the other RTC-S1 layers. Then at the end we concatenate all burst logs into the log file that the user set in the CLI command.

I think we can visit this topic after we address other tasks for the delivery?